debezium-core/src/main/java/io/debezium/transforms/debezium-timestamp-converter/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That POM is not really needed here, as this is part of the _debezium-core_ module now.